pkg/database/database.go

```diff
@@ -147,11 +150,13 @@ type replicaState struct {
 
 // IDB database instance
 type db struct {
-	st *store.ImmuStore
+	st          *store.ImmuStore
+	objectStore *store.ImmuStore
```

Comment:
@jeroiraz should we have a separate store for objects, or should there be a single store? If we use a single store, the starting tx id becomes 3, instead of 2 because there is one more commit for document store